### ISSUE TITLE
'release-0*' deployments don't set imagePullPolicy to Always

### ISSUE THREAD

**Issue by mangelajo** [submariner-io/submariner-operator#1131]
**What happened**:

You can grab an outdated "release-0.x" image, and without pullPolicyAlways
an updated image won't be loaded.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

1. Download release-0.8 subctl: `curl -Ls https://get.submariner.io | VERSION=release-0.8 bash`

2. Deploy Submariner via subctl

3. The expected behavior is that the system will pull the most recent images, but in practice the images being pulled are outdated. For e.g, for lighthouse-agent I would expect it to use https://quay.io/repository/submariner/lighthouse-agent/manifest/sha256:90d4d760c5cd3abd226f7849e7ca43154bf35f712528254af215c36b76f39f75 (from 5 days ago as of March 9), but it actually uses the one tagged with 0.8.1. To fix this, Deployments should use `imagePullPolicy` = "Always" by default

**Anything else we need to know?**:

To workaround this, it possible to "disable" the operator (change replicas to 0), than manually edit the Deployment YAML and set the imagePullPolicy  to "Always". But it's not very friendly nor sustainable. 

In other words, `release-0.8` is really the "devel" of the 0.8 branch, so we should always pull the most up to date code.

**Environment**:
- Submariner version (use `subctl version`):
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by nyechiel** [comment]
More info on this and steps to reproduce:

1. Download release-0.8 `subctl`:

curl -Ls https://get.submariner.io | VERSION=release-0.8 bash

2. Deploy Submariner via `subctl`

3. The expected behavior is that the system will pull the most recent images, but in practice the images being pulled are outdated. For e.g, for lighthouse-agent I would expect it to use https://quay.io/repository/submariner/lighthouse-agent/manifest/sha256:90d4d760c5cd3abd226f7849e7ca43154bf35f712528254af215c36b76f39f75 (from 5 days ago as of March 9), but it actually uses the one tagged with 0.8.1. To fix this, Deployments should use `imagePullPolicy` = "Always" by default
